salt/utils/napalm.py

```diff
@@ -316,7 +316,9 @@ def get_device(opts, salt_obj=None):
         try:
             provider_lib = importlib.import_module(network_device.get('PROVIDER'))
         except ImportError as ierr:
-            log.error('Unable to import {0}'.format(network_device.get('PROVIDER')), exc_info=True)
+            log.error('Unable to import %s',
+                      network_device.get('PROVIDER'),
+                      exc_info=True)
```

Comment:
Just FYI, `log.error('something', exc_info=True)` and `log.exception('something')` are identical.

https://docs.python.org/2/library/logging.html#logging.exception

I don't think it necessarily needs to be changed though, this is just an FYI.